lms/static/scripts/frontend_apps/common/dom.js

```diff
@@ -2,9 +2,10 @@
  * Attach listeners for one or multiple events to an element and return a
  * function that removes the listeners.
  *
- * @param {Element}
+ * @param {Element} element
  * @param {string[]} events
  * @param {(event: Event) => any} listener
```

Comment:
not that we need to change this, but I have been going about using generic param names when typing a function. Does anyone have a preference

 `@param {(event: Event) => any} listener`
vs
` @param {(a: Event) => any} listener`

Reply:
I'm leaning towards single-letter parameter being preferred in function types where they don't provide any information that is not obvious from the types themselves. In that case the parameter name is really only there to satisfy the grammar.

The only change I would make is to your second suggestion is to use the same letter that the type name starts with (ie. `@param {(e: Event) => any} listener`). That's just an aesthetic preference on my part though.